### ISSUE TITLE
Question about the training hyperparameters （epoch/lr)

### ISSUE THREAD

**Issue by skylineo** [KexianShen/trajectory-prediction#1]
Thank you for your work, this is an excellent piece of code. I’ve recently been trying to apply different pretraining strategies to trajectory prediction networks. In your code, I noticed that the pretraining epoch is set to 20, the fine-tuning epoch is set to 40, and the learning rate is the same for both (ignoring the differences due to cosine decay). However, in other papers, I’ve found that in some cases, the pretraining epochs and learning rates are much larger than for fine-tuning (e.g., SEPT, TrajMAE), or both parts use the same hyperparameters (e.g., Forecast-MAE). Currently, I’m planning to modify your code. Could you advise on how to set the pretraining and fine-tuning epochs and learning rates more reasonably？Is 20 epochs for pretraining and 40 epochs for fine-tuning a reasonable choice?

**Comment by KexianShen** [comment]
I didn't think much about the epoch and learning rate at the time, as the pre-training results were quite good. Your suggestion is excellent. 

Wishing you better results in your experiments.

**Comment by skylineo** [comment]
Got it! Thanks for your timely response!
